### ISSUE TITLE
nice system..

### ISSUE THREAD

**Issue by lordmen99** [asier13/Python-Trading-Bot#1]
hi i see ur trade rsi system looks good, but have you try in real trade? how the result?

**Comment by asier13** [comment]
Hi,
I did try the strategy with real trades, it has problems calculating the quantity amount when it triggers the TPs. When it was time to TP or Sell(SL) it only executed a very small amount of the quantity bought previously. I have plans to resume coding and improving the functionality, right now I have no time but maybe in 3-4 months Im able to do so. I think this version is not the last one I have on local, if you want I can upload it so you can take a look. Other than that, the strategy did work, I tried for several days leaving the bot run and if the TPs worked well I would have made profit for sure.

**Comment by lordmen99** [comment]
yes, try upload to here. so, per trade its how much profit can get in %?

**Comment by asier13** [comment]
The average is 0.7 to 1% per trade but you make around 2 trades a day. I will upload the latest code I have next week cuz right now I dont have access to the pc that has the files.

**Comment by z41nl** [comment]
upload pnl from ur exchange plz
